nh_eobs_mobile/static/src/js/nhlib.js

```diff
@@ -1226,7 +1230,12 @@ NHMobilePatient = (function(superClass) {
         chartFunc(self, serverData);
       }
       if (validTable) {
-        return tableFunc(self, serverData);
+        tableFunc(self, serverData);
+      }
+      if (activeTab === "#graph-content") {
```

Comment:
Haha oh dear, that makes sense, I was getting confused with `Object.is(arg1, arg2)`, cool :+1: